### ISSUE TITLE
[3주차] Deep Clustering  for Unsupervised Learning of Visual Features 

### ISSUE THREAD

**Issue by hs1003** [jwkanggist/self-supervised-learning-narratives-1#9]
# Keywords
unsupervised learning, clustering

# TL;DR

# Abstract
Clustering is a class of unsupervised learning methods that has been extensively applied and studied in computer vision. Little work has been done to adapt it to the end-to-end training of visual features on large scale datasets. 
In this work, we present DeepCluster, a clustering method that jointly learns the parameters of a neural network and the cluster assignments of the resulting features. DeepCluster iteratively groups the features with a standard clustering algorithm, kmeans, and uses the subsequent assignments as supervision to update the weights of the network. We apply DeepCluster to the unsupervised training of convolutional neural networks on large datasets like ImageNet and YFCC100M. The resulting model outperforms the current state of the art by a significant margin on all the standard benchmarks. 

# Paper link
https://arxiv.org/pdf/1807.05520.pdf

# Presentation link
https://docs.google.com/presentation/d/1gtgCMdUENK_kIeHd5y6l-P2_8KfnYJCdjHYyY2Koivo/edit?usp=sharing

# video link
